modules/programs/alot-accounts.nix

```diff
@@ -13,6 +13,29 @@ with lib;
       '';
     };
 
+    contactCompletion = mkOption {
+      type = types.attrsOf types.str;
+      default = {
```

Comment:
I'm a bit worried how this default value will look in the manual. In particular, it would be bad if the `pkgs.notmuch` reference becomes evaluated when the module is read in.

Reply:
Perhaps a safer default would be `null` so that it becomes the user's responsibility to set it?

Reply:
I've used `defaultText` to override the value displayed in the manual.
The default doesn't require anything extra, it just makes things work out of the box, which is a good thing IMO.